### PR TITLE
Rechart - Task Info Card

### PR DESCRIPTION
**Pull request by ShevchukTania**
**STATUS**
WIP

**DESCRIPTION**

**CHECK LIST**
- [ ] specs added
- [ ] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [ ] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
image

### CODE REVIEW

File: app/javascript/bundles/Dashboard/components/TasksInfoCard/TasksInfoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        <Line type='monotone' dataKey='due soon' stroke='#ff7300'/>
+        <Line type='monotone' dataKey='outdated' stroke='#ff4306'/>
+      </ComposedChart>
+    );
+  }
+}
+
+class SelectWorkspace extends React.Component {
+
+	render () {
```
Comment (baarento) [inline app/javascript/bundles/Dashboard/components/TasksInfoCard/TasksInfoCard.jsx]: відступи

File: app/controllers/account/dashboard_controller.rb [hunk 2]
```diff
@@ -5,15 +5,28 @@ def index
...
 
   def top_workspaces_card
-
   end
-  
-  def user_info_card
 
-  end
+  def tasks_info
+    @task_info = []
```
Comment (igorkasyanchuk) [inline app/controllers/account/dashboard_controller.rb]: move logic from controller to new class

app/lib/TaskInfoReport and pass current_user and return hash

and in controller

render json: { info: TaskInfo.new(current_user).report, workspaces: WorkspacesReport.new(current_user).report }

File: app/javascript/bundles/Dashboard/components/SelectWorkspace/SelectWorkspace.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./select-workspace.scss";
+
+class SelectWorkspace extends React.Component {
+	render () {
```
Comment (igorkasyanchuk) [inline app/javascript/bundles/Dashboard/components/SelectWorkspace/SelectWorkspace.jsx:6]: fix alignment

File: app/controllers/account/dashboard_controller.rb [hunk 4]
```diff
@@ -5,15 +5,11 @@ def index
...
-
-  end
-  
-  def user_info_card
-
   end
 
-  def tasks_info_card
-
+  def tasks_info
```
Comment (igorkasyanchuk) [inline app/controllers/account/dashboard_controller.rb:15]: where is spec? at least something simple to make sure code works

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**